ui/react-components/containers/AppointmentContainer.jsx

```diff
@@ -16,15 +16,15 @@ class AppointmentContainer extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            locale: getLocale(),
+            locale: getLocale() === 'pt_BR' ? 'pt': getLocale(),
             messages: translations[props.locale],
             appConfig: null
         };
         (async () => {
-            this.setState({messages: await getMessages(this.state.locale)});
+            this.setState({messages: await getMessages(getLocale())});
             this.setState({appConfig: await getAppConfig()});
         })();
-        moment.locale(getLocale());
+        moment.locale(getLocale() === 'pt_BR' ? 'pt': getLocale());
```

Comment:
@supriyatw should it be just 'pt' or 'pr-BR' ?

Reply:
Thanks for pointing out Bindu. React Intl is not suppporting the key pt_BR but supporting pt-BR made required changes can you check once